aries-site/src/pages/components/checkbox.mdx

```diff
@@ -36,8 +37,22 @@ Checked and unchecked states refer to when an individual CheckBox is checked or
 
 ### CheckBox outside of FormField
 
+When using CheckBox outside of Formfield, it does not have a border or label. The hover indentifier will also be on the CheckBox itself.
+Here are some common use cases for when a CheckBox can be used outside of a FormField:
```

Comment:
Only capitalize when we are referencing the actual component (proper noun). Sentence case when writing about the concept.

```suggestion
A checkbox might be used outside of a form field when:
```